From 94bda3e1781e8fa82074c80f9d21ad632fa24f92 Mon Sep 17 00:00:00 2001
From: Shuguang Gong <shuguang.gong@intel.com>
Date: Mon, 4 May 2015 15:09:50 +0800
Subject: [PATCH 382/462] atomisp2: add edge enhancement interface for SoC
 sensor

Change-Id: Ib4c02f9e19e638eb592e5151dd895011f1134773
Tracked-On:https://jira01.devtools.intel.com/browse/IMINAN-34925
Signed-off-by: Shuguang Gong <shuguang.gong@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/54153
Validation-Android: Wei Tao <wei.tao@intel.com>
Approver: Jing Tao <jing.tao@intel.com>
Integrator: WeiX Meng <weix.meng@intel.com>
Build: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
Maintainer: WeiX Meng <weix.meng@intel.com>
---
 .../camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_ioctl.c     | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_ioctl.c b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_ioctl.c
index c07877e..bcfd8f4 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_ioctl.c
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_ioctl.c
@@ -2729,6 +2729,7 @@ static long atomisp_vidioc_default(struct file *file, void *fh,
 	case ATOMISP_IOC_S_SENSOR_AE_BRACKETING_MODE:
 	case ATOMISP_IOC_G_SENSOR_AE_BRACKETING_MODE:
 	case ATOMISP_IOC_S_SENSOR_AE_BRACKETING_LUT:
+	case ATOMISP_IOC_S_SENSOR_EE_CONFIG:
 		/* we do not need take isp->mutex for these IOCTLs */
 		break;
 	default:
-- 
1.9.1

